### DISCUSSION TITLE
Add linux arm64 support

### DISCUSSION THREAD

**Discussion by thenameisluk** [ppy/osu#28488] [Ideas] [answered]
it is already possible to [compile osu for linux arm64](https://github.com/hexdump0815/imagebuilder/issues/217)
and it works fine
![obraz](https://github.com/ppy/osu/assets/92756992/2b94b41e-0a26-4c19-8e55-6b0f27099e0c)
but it is still a development build
and these notifications are getting pretty annoying
![obraz](https://github.com/ppy/osu/assets/92756992/c15f7e35-46b8-4872-8500-b0b4dd01bc6c)
adding an arm64 build would add ability to play for
- m1 m2 ... macs that run asahi linux
- arm64 chromebooks
- raspberry pi
- and other arm64 linux devices

there have been [attempts](https://github.com/ptitSeb/box64/issues/829) to run osu via box64 but it's really hard to setup and not only runs very poorly performance wise but also is very unstable, it doesn't really make sense to do that if you can provide a native version

and since it doesn't really look that hard to compile for arm64
simply adding appimage would solve an issue and shouldn't be a lot of work

**Comment by Joehuu** [comment, marked as answer]
Progress has been made https://github.com/ppy/osu-deploy/pull/170.
